wordpress-mobile/WordPress-iOS
# [SIWA] Show email address on WP password entry view

Fixes #12556 
WPKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/183
WPAuth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/141

This simply updates WPKit and WPAuth pods to include the fix for displaying the email address on the WP password entry view.

To test:

To note, I only saw the bug when using Touch ID to confirm using my Apple ID.

![touch_id](https://user-images.githubusercontent.com/1816888/66001191-3b08c500-e45e-11e9-86c2-29171eba05da.jpeg)

- Login/signup with SIWA using an email that matches a WP account.
- If prompted for your password, log out and login with SIWA again.
- On the WP password entry view, verify the email address is now displayed.

![password_entry](https://user-images.githubusercontent.com/1816888/66001286-5f64a180-e45e-11e9-85b7-9595f9663161.jpeg)

Update release notes:
